mongodb/migrations/migrations_test.go

```diff
@@ -183,6 +183,176 @@ func TestMigrations_m004(t *testing.T) {
 	assert.Nil(t, key2["path"])
 }
 
+func TestMigrations_m005(t *testing.T) {
+	ctx := context.Background()
+	db := setup(t, ctx)
+	_, err := db.Collection("bucketarchives").InsertMany(ctx, []interface{}{
```

Comment:
Tried to insert many different permutations and combinations of data here to make sure we test all possibilities.